addons/event_booth/models/event_booth.py

```diff
@@ -77,14 +77,16 @@ def create(self, vals_list):
         return res
 
     def write(self, vals):
+        to_confirm = self.filtered(lambda booth: booth.state == 'available')
         wpartner = {}
         if 'state' in vals or 'partner_id' in vals:
             wpartner = dict(
                 (booth, booth.partner_id.ids)
                 for booth in self.filtered(lambda booth: booth.partner_id)
             )
 
-        res = super(EventBooth, self).write(vals)
+        write_vals = {field: value for field, value in vals.items() if field in self._fields}
```

Comment:
Not sure, why filtering on self._fields ?

Edit: I see in website event booth exhibitor we use keys like sponsor_something to filter them out and create the sponsor. Won't that be broken with this filtering ?